web/app/themes/dp-timber-theme/.storybook/manager.js

```diff
@@ -0,0 +1,41 @@
+import { addons } from "@storybook/manager-api";
+import { create } from "@storybook/theming";
+
+addons.setConfig({
+  theme: create({
+    base: "light",
+
+    brandTitle: "My custom Storybook",
+    brandUrl: "https://digitalpolygon.com",
+    brandImage: "https://www.digitalpolygon.com/DigitalPolygon-logo.svg",
+    brandTarget: "_self",
+
+    //
+    colorPrimary: "#3A10E5",
```

Comment:
can you also explain what all these variable do ?

Reply:
These variable defined by the storybook to change the UI color of the Storybook, If variable name changes the changes will not work